code/__HELPERS/_macros.dm

```diff
@@ -39,7 +39,7 @@
 
 #define iscluwne(A) istype(A, /mob/living/simple_animal/hostile/retaliate/cluwne)
 
-#define isAI(A) istype(A, /mob/living/silicon/ai)
+#define isAI(A) (istype(A, /mob/living/silicon/ai) || (isobserver(A) && ticker.current_state == GAME_STATE_FINISHED)) 
```

Comment:
Have you gone through every single instance of isAI() to see if it will still maintain the proper behavior? Otherwise, no, you cant make this kind of large scale change because you cant possibly predict every way in which it could go wrong.